themes/bootstrap3/templates/RecordTab/holdingsils.phtml

```diff
@@ -12,7 +12,12 @@
     try {
         $holdings = $this->driver->getRealTimeHoldings();
     } catch (\VuFind\Exception\ILS $e) {
-        $holdings = ['holdings' => []];
+        $holdings = [
+          'holdings' => [],
+          'total' => 0,
```

Comment:
Should we initialize electronic_holdings here too?

Reply:
It's checked with !empty() but it doesn't hurt to initialize it here too.